lavalink/client.py

```diff
@@ -69,9 +69,10 @@ def add_event_hook(self, hook, event: Event = None):
             dispatched.
         """
         event_hooks = self._event_hooks.get(event)
+
         if hook not in event_hooks:
```

Comment:
If `event_hooks` is None, this will throw:

![](https://serux.pro/f0c285b3a7.png)